src/views/domain-workflows/domain-workflows-query-label/domain-workflows-query-label.styles.ts

```diff
@@ -68,6 +68,13 @@ export const styled = {
 
 export const overrides = {
   popover: {
+    Body: {
+      style: ({ $theme }) => ({
+        marginLeft: $theme.sizing.scale600,
```

Comment:
this can be replaced with `popoverMargin` prop

Reply:
Why are we ignoring the bottom?

Reply:
I wanted to prevent the popover from becoming smaller on small screens. I'll test this out to see if it has any problems.

Reply:
Made it a full margin of 16px on all sides. `popoverMargin` only sets the top margin 🥲